.expeditor/verify.pipeline.yml

```diff
@@ -289,6 +289,18 @@ steps:
       executor:
         docker:
 
+  - label: "[unit] e2e"
```

Comment:
OK, it is weird calling it "unit" tests... except that's where we have been putting linting (as prior ones have been unit-plus-lint). Could just call this one `[static]` I suppose. Other ideas...?

Reply:
We used to differentiate `[static]` and `[unit]` for services, e.g. authz-service, when we still ran them separately. When they were merged (because there wasn't that much value in having them separate, and a gigantic number of tests is hard to work with, I guess), static became part of unit. So yeah, let's  call this `[static]` 😃